doc/articles/features/using-skia-gtk.md

```diff
@@ -57,9 +57,50 @@ If you want to upgrade **SkiaSharp** to a later version, you'll need to specify
 
 ```xml
 <ItemGroup>
-   <PackagReference Include="SkiaSharp" Version="2.88.0" />
-   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.0" />
-   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.0" />
-   <PackageReference Update="SkiaSharp.NativeAssets.macOS" Version="2.88.0" />
+   <PackagReference Include="SkiaSharp" Version="2.88.3" />
+   <PackagReference Include="SkiaSharp.Harfbuzz" Version="2.88.3" />
+   <PackagReference Include="SkiaSharp.NativeAssets.Linux" Version="2.88.3" />
+   <PackageReference Update="SkiaSharp.NativeAssets.macOS" Version="2.88.3" />
 </ItemGroup>
 ```
+
+### .NET Native AOT support
+
+Building an Uno Platform Skia+GTK app with .NET (7+) Native AOT requires, GtkSharp 3.24.24.38 (or later), or Uno Platform 4.7 (or later).
+
+To build an app with this feature enabled:
+1. Add the following property in your `.csproj`:
+   ```xml
+   <PropertyGroup>
+   		<PublishAot>true</PublishAot>
+	</PropertyGroup>
+   ```
+1. Add the following items in your `.csproj`:
+   ```xml
+   <ItemGroup>
+		<RdXmlFile Include="rd.xml" />
```

Comment:
I would suggest name file as `Uno.rd.xml` that way you can combine multiple rd.xml files in same app. That's easier to manage when you have multiple technologies which require customizations. At least this is my idea behind https://github.com/kant2002/rdxmlLibrary/

Reply:
Great suggestion! Thanks for the review.

For what's required in GtkSharp at this point, I'm not sure and that's why it's fully included in the sample as it's more of an abundance of caution. I'd rather have to deal with individual issues as they arise while trimming down.

Reply:
If you can point me how I can get GtkSharp native dependencies in easy way, I can provide suggestion in exact types which have to be rooted. No need to drag whole assembly fro that. As alternative you may land this PR for example and I can work on improving things. That way you have “something” which is surely better then perfect something from future

Reply:
Will do for sure. Let's see in https://github.com/unoplatform/uno/issues/10355 it once this PR gets merged.